data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1158,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
```

Comment:
Since you're also nerfing the Peregrine, you should include its name in the title of the PR.